src/addtodo.js

```diff
@@ -0,0 +1,67 @@
+
+class AddTask {
+  constructor(title, description, ddate, priority) {
+    this.title = title;
+    this.description = description;
+    this.ddate = ddate;
+    this.priority = priority;
+    this.id = Date.now().toString();
+  }
+}
+
+const title = document.querySelector('#title');
+const description = document.querySelector('#description');
+const ddate = document.querySelector('#date');
+const priority = document.querySelector('#priority');
+const todoindex = document.querySelector('#todoindex');
+const submit = document.querySelector('.submit');
+const modal = document.querySelector('#myModal');
+const newTaskForm = document.querySelector('[data-new-task-form]');
+let modalOpen = false;
```

Comment:
- Keep DOM manipulation in a separate file/class from this class. This class should only contain business logic. Expect to see only the `addTodo(Todo todo) and editTodo(Todo todo)` functions. So create a `dom_logic.js` file in the `src/` folder for dom specific logic and move any dom related code there.

Reply:
Thanks, @whiz25 for The review. We removed those to a separate file. 👍